test/unit/kubernetes-deploy/kubernetes_resource/stateful_set_test.rb

```diff
@@ -40,7 +40,7 @@ def build_synced_ss(template:)
     sync_mediator.kubectl.expects(:run).with("get", "Pod", "-a", "--output=json", anything).returns(
       ['{ "items": [] }', "", SystemExit.new(0)]
     )
-    sync_mediator.kubectl.expects(:server_version).returns(Gem::Version.new("1.8"))
+    sync_mediator.kubectl.expects(:server_version).returns(Gem::Version.new(KubernetesDeploy::MIN_KUBE_VERSION))
```

Comment:
I think neither Deployment nor StatefulSet need to be actually grabbing the `version` during sync anymore. Looks like that should have been removed when we deprecated 1.7.